arangodb-net-standard/IndexApi/Models/InvertedIndexStoredValue.cs

```diff
@@ -2,9 +2,31 @@
 
 namespace ArangoDBNetStandard.IndexApi.Models
 {
+    /// <summary>
+    /// Stored value for inverted index.
+    /// </summary>
     public class InvertedIndexStoredValue
     {
+        /// <summary>
+        /// A list of attribute paths. 
+        /// The . character denotes sub-attributes.
+        /// </summary>
         public IEnumerable<string> Fields { get; set; }
+
+        /// <summary>
+        /// Optional. Defines how to compress the 
+        /// attribute values. Possible values:
+        /// "lz4" (default): use LZ4 fast compression.
+        /// "none": disable compression to trade space for speed.
```

Comment:
Minor improvement suggestion: Could be represented as constants in this class or in a separate class, e.g. `CompressionTypes`.

We also have similar constants in `ArangoDBNetStandard.ViewApi.Models.ViewStoredValue`, but maybe it's better to keep the concerns separate?

Reply:
Done.